### DISCUSSION TITLE
Standup (Wednesday, May 11, 2022)

### DISCUSSION THREAD

**Discussion by antonieto** [CarlosCRG19/ijalti-peers#55] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:
```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
```

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - Not much

**What are your priorities for the next 24 hours?**:
 - Find some time to start documenting endpoints

**Blockers**:
 - Some stuff from work 😣

**Shoutouts**:
 - @paotg123

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Cleaned up some code and restructured folders for front-end with @CarlosCRG19 

**What are your priorities for the next 24 hours?**:
 - Make adjustments for handling forms

**Blockers**:
 - None

**Shoutouts**:
 - @CarlosCRG19 for being such a good teacher 😯

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
 - Watched a couple of tutorials
 
**What are your priorities for the next 24 hours?**:
 - idk

**Blockers**:
 - Actually doing something on this shitty Semana Tec (sorry for venting)

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 -  Not much

**What are your priorities for the next 24 hours?**:
 -  Start documenting endpoints

**Blockers**:
 -  Lots of homework

**Shoutouts**:
 - @CarlosCRG19  que está pisteando
